app/connectors/EmailVerificationConnector.scala

```diff
@@ -33,15 +33,15 @@ import scala.concurrent.Future
 class EmailVerificationConnector @Inject()(http: HttpClient,
                                            appConfig: AppConfig) {
 
-  private[connectors] def checkVerifiedEmailUrl(email: String): String =
-    s"${appConfig.emailVerificationBaseUrl}/email-verification/verified-email-addresses/$email"
+  private val checkVerifiedEmailUrl: String =
```

Comment:
Does this need to be left as def so we have the flexibility of being able to change the url in config without having to redeploy the app?

Reply:
App-config changes always require a redeploy to take affect, the app-config is downloaded and bundled with the release, isn't it?

Reply:
Thanks that's good to know :)